extension/encoding/awscloudwatchmetricstreamsencodingextension/extension_test.go

```diff
@@ -26,7 +26,7 @@ func TestNew_OpenTelemetry10(t *testing.T) {
 	require.NotNil(t, e)
 
 	_, err = e.UnmarshalMetrics([]byte{})
-	assert.EqualError(t, err, `UnmarshalMetrics unimplemented for format "opentelemetry1.0"`)
+	assert.EqualError(t, err, errUvarintReadFailure.Error())
```

Comment:
We should probably make this a bit more descriptive, e.g. `failed to unmarshal metrics as "opentelemetry1.0" format: %w`